docs/source/endpoints/translations.md

```diff
@@ -128,3 +128,12 @@ This endpoint returns the proper placement for the newly created translation:
 This endpoint can be used with the {doc}`expansion` mechanism which allows getting additional information about a content item in one query, avoiding unnecessary requests.
 
 If a simple `GET` request is done on the content item, a new entry will be shown on the `@components` entry, with the URL of the `@translations` endpoint:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/translations_expand_get.req
+```
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/translations_expand_get.resp
+:language: http
+```
```

Comment:
Add here:
```
A request to provide the content information plus the translations information can be down with
'/plone/en/test-document?expand=translations'
This expands the 'translations'  part of '@components'.
```

TODO Generate a response with translations in test. Then add request and response here.